renku/core/management/workflow/providers/toil.py

```diff
@@ -45,6 +45,13 @@
 from renku.core.plugins import hookimpl
 from renku.core.plugins.provider import RENKU_ENV_PREFIX
 
+# silence cwltool version warning
+logger = logging.getLogger("toil.cwl")
+logger.disabled = True
+from toil.leader import FailedJobsException  # noqa
+
+logger.disabled = False
```

Comment:
Do we need to set disabled twice?

Reply:
well i though that maybe having it on again would be not a bad idea :) but i'll close this as pinning of `cwltool` fixes the whole problem that this supposed to be solving temporarily